package.json

```diff
@@ -25,7 +25,8 @@
     "react-dom": "^18.2.0",
     "react-loading-skeleton": "^3.1.0",
     "server-only": "^0.0.1",
-    "sharp": "0.31.1"
+    "sharp": "0.31.1",
+    "v8": "^0.1.0"
```

Comment:
why did we have to install v8?

Reply:
There was some error that kept popping up with something i was trying to implement that kept prompting me to install it